core/src/main/java/com/linecorp/armeria/common/logging/DefaultRequestLog.java

```diff
@@ -1811,6 +1565,13 @@ public List<RequestLogAccess> children() {
             return DefaultRequestLog.this.children();
         }
 
+        @Override
+        public Set<RequestLogProperty> availableProperties() {
+            return Sets.immutableEnumSet(Arrays.stream(RequestLogProperty.values())
+                                               .filter(this::isAvailable)
+                                               .collect(toImmutableSet()));
```

Comment:
`isAvaliable` always returns true in `CompleteRequestLog`.
How about adding `RequestLogProperty.allProperties()` and returning it here?